src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/bubbleVisualization.js

```diff
@@ -0,0 +1,40 @@
+class BubbleVisualization extends Visualization {
+    static inputType = "Any"
+
+    onDataReceived(data) {
+        this.setPreprocessor("None");
```

Comment:
this line should not be needed and can have performance issues - please confirm it with @farmaazon and @iamrecursion

Reply:
Will do!

Reply:
Agreed with @farmaazon to remove this line, this is dead code.